### ISSUE TITLE
publications: add MuMMi SC19 paper

### ISSUE THREAD

**Issue by SteVwonder** [flux-framework/flux-framework.github.io#51]

**Comment by dongahn** [comment]
We may also want to add those recently accepted papers without PDFs for now.
1. Generalized MuMMI for SC21
2. @tpatki's IEEE Cluster power analysis paper.

**Comment by vsoch** [comment]
hey @dongahn @SteVwonder could you provide links to these exact papers? I'd be happy to add them.

**Comment by dongahn** [comment]
https://ieeexplore.ieee.org/document/9910090

https://ieeexplore.ieee.org/document/9973491

**Comment by vsoch** [comment]
Thank youuuu!

**Comment by dongahn** [comment]
BTW I briefly looked at your `flux-operator` work and that's awesome! I can't wait to test it!

**Comment by vsoch** [comment]
Thank you! I made a [tool over break](https://converged-computing.github.io/flux-cloud/) to make it easy to bring up a cluster, install the operator, run a grid of experiments, and bring everything down - I had a PR in today to add AWS to that, and should have it fully done by end of  day (I first used the command line client for eksctl and then promptly learned it didn't support a networking option I will need, badoom fzzzz!)

Please see https://github.com/flux-framework/flux-framework.github.io/pull/70 for this issue.

**Comment by dongahn** [comment]
Wow! You should do a tutorial video for all these new shiny tools -- very promising!

**Comment by vsoch** [comment]
Yeah, I totally could do that! Right now I'm just doing asciinemas. It would have been cool to write in Go but I already wrote cloud-select (which could help with prices) in Python so that determined that decision.

**Comment by dongahn** [comment]
BTW, i have microk8s running on an ec2 instance (`ubuntu 20.04`). I can just deploy flux operator and play with it correct? No known issues?

**Comment by vsoch** [comment]
Oh please do try! I haven't ever tried microk8s (but I assume it's similar in spirit to minikube or kind?) For the first, I had to enable DNS/ingress (see here https://flux-framework.org/flux-operator/development/developer-guide.html#setup) and I wonder if there is something similar for microk8s.

And I'm not totally happy with these docs yet, but I just added a "quick production deploy" section that works for a production cluster (so far has worked with kubetl via gcloud cluster create and eksctl create cluster), those instructions are here: https://flux-framework.org/flux-operator/getting_started/user-guide.html#production. I'm going to further clean that up, but probably not today!

And the best set of dummy examples (so far) are in examples/tests and examples/flux-restful (for the interface). https://github.com/flux-framework/flux-operator/tree/main/examples.

If microk8s requires something special let's document it and be sure to add a guide. If you don't have time I can also try installing and using it. Probably the most complexity will be with respect to how these different tools handle local volumes. 

Finally, the flux-cloud aws work should be merged later this evening - I'm finishing up dummy experiments, and tomorrow I'll start some of our more scaled ones (and do a release after that).

**Comment by vsoch** [comment]
@SteVwonder @dongahn you are both authors on one/more of these papers - it looks like we aren't technically allowed to post the PDF from IEEE (as I've done in the PR) but need to get the draft that was submit. Could you send these to me?

**Comment by SteVwonder** [comment]
I don't.  I believe there is an internal repo somewhere in LLNL that has them.  I'll ping @dongahn via email to see if he has a copy of either.

**Comment by SteVwonder** [comment]
Actually. The MuMMi SC21 paper was in overleaf.  So here is a copy of that 
[MuMMI_SC2021.pdf](https://github.com/flux-framework/flux-framework.github.io/files/10803549/MuMMI_SC2021.pdf).

I see the scalable composition paper in overleaf too, but I'll let Dong vouch for if that is the latest version or not and post it here.
